pkg/extensions/tbtc/tbtc.go

```diff
@@ -934,31 +938,31 @@ func (t *tbtc) waitDepositStateChangeConfirmation(
 }
 
 func (t *tbtc) waitKeepNotActiveConfirmation(
-	keepAddress string,
+	keep chain.BondedECDSAKeepHandle,
 ) bool {
-	currentBlock, err := t.chain.BlockCounter().CurrentBlock()
+	currentBlock, err := t.blockCounter.CurrentBlock()
 	if err != nil {
 		logger.Errorf(
 			"could not get current block while confirming "+
 				"keep [%v] is not active: [%v]",
-			keepAddress,
+			keep.ID(),
```

Comment:
Same problem in logs.

```suggestion
			keep.ID().String(),
```

Reply:
Resolving as per https://github.com/keep-network/keep-ecdsa/pull/714#discussion_r586510627